### DISCUSSION TITLE
优选IP是安装首次才有，还是运行过程会自动更新？

### DISCUSSION THREAD

**Discussion by home2020** [vvbbnn00/WARP-Clash-API#123] [General]
优选IP是安装首次才有，还是运行过程会自动更新？

**Comment by vvbbnn00** [comment]
可以配置环境变量以实现定时选优：https://github.com/vvbbnn00/WARP-Clash-API?tab=readme-ov-file#-%E7%8E%AF%E5%A2%83%E5%8F%98%E9%87%8F

也可以通过执行命令来手动选优：https://github.com/vvbbnn00/WARP-Clash-API?tab=readme-ov-file#-%E6%89%8B%E5%8A%A8ip%E9%80%89%E4%BC%98
